### PR TITLE
feat(Export.XML): export to XML format readable by Excel

### PR DESCRIPTION
**Pull request by gitys**
Minor feature: option to output column headers.

### CODE REVIEW

File: doc/help/index.html [hunk 1]
```diff
@@ -1381,6 +1381,17 @@
             single CSV file.
         </p><a href="#top">Back to Top</a>
         <hr>
+        <h2>Exporting to XML Files</h2>
+        <p>
+            MMEX can export to an XML file. The XML forma is compliant with
```
Comment (jgen) [inline doc/help/index.html]: Minor typo: Should "forma" should be "format" ?

Comment (gitys) [inline doc/help/index.html]: Thanks. Fixed.

**Comment by vomikan** [comment]
Bug found. If I want just export some account to XML file I cant becouse a file missing.
Other words on export the file should be created (or owerwriten).

**Comment by vomikan** [comment]
Also my Excel does not support values >= 1000.00
~~~~~
Ошибка XML в "Таблица"
Причина:	Ошибочное значение
Файл:	C:\Temp\test_xml.xml
Группа:	Cell
Тег:	Data
Значение:	-1 182.00

Ошибка XML в "Таблица"
Причина:	Ошибочное значение
Файл:	C:\Temp\test_xml.xml
Группа:	Cell
Тег:	Data
Значение:	28 173.00

Ошибка XML в "Таблица"
Причина:	Ошибочное значение
Файл:	C:\Temp\test_xml.xml
Группа:	Cell
Тег:	Data
Значение:	-5 852.00

Ошибка XML в "Таблица"
Причина:	Ошибочное значение
Файл:	C:\Temp\test_xml.xml
Группа:	Cell
Тег:	Data
Значение:	-2 500.00

Ошибка XML в "Таблица"
Причина:	Ошибочное значение
Файл:	C:\Temp\test_xml.xml
Группа:	Cell
Тег:	Data
Значение:	-1 667.00
~~~~~

**Comment by gitys** [comment]
I will check these points.
Thanks.

**Pull request merged**